indexing-hadoop/src/main/java/io/druid/indexer/partitions/HashedPartitionsSpec.java

```diff
@@ -50,4 +51,12 @@ public Jobby getPartitionJob(HadoopDruidIndexerConfig config)
   {
     return new DetermineHashedPartitionsJob(config);
   }
+
+  @Override
+  @JsonProperty
+  @Nullable
+  public List<String> getPartitionDimensions()
+  {
+    return null;
```

Comment:
How would you feel about making this passed as a json property in HashedPartitionsSpec and add it to the docs, so that people can use it ?